blips/blip-3-peerreview.md

```diff
@@ -27,27 +29,37 @@ This standard should specify which fields to register in every record, the forma
 
 
 - [ ] **TODO** Decide which field could be private. For instance, the review content could be publicly shared or not depending on the reviewer's preferences.
 ### Existing standards (industry / de-facto / formalized)
 
 - [ ] **TODO** Write a collection of existing standards: Crossref, Orcid, Publons, https://casrai.org/ (ORCid uses casrai, they are preparing a ISO standard on that standard proposal), etc.
 
+- Orcid: https://support.orcid.org/hc/en-us/articles/360006971333-Peer-Review
+- Crossref: https://www.crossref.org/education/content-registration/recommendations-by-content-type/peer-reviews/
+
+
+## Functions and Permissions
+
+
+| Function name | description | parameters | permission |
+| -------- | -------- | -------- |  -------- |
+| addReview | add a review to the system     | author, journalId, publisher, manuscriptId, ManuscriptHash, recommendation, url    | reviewers can add reviews to their profile. **TODO:** consider other authorized actors such as journals or review registering services (Publons, Orcid,...) |
+- [ ] **TODO** finish previous function specification table
 
-## Decentralized Vouchers for Review Validity
-We should allow other reviewers and third parties to vouch for the validity of reviews registered at the system.
+## Decentralized Endorsements for Review Validity
+We should allow other reviewers and third parties to endorse for the validity of reviews registered at the system.
 
-Different actors can have different strategies to verify a review. These actors can register their addresses as verifiers, and register a URL explaining how they validate the reviews. For instance, Bloxberg peer-review-app can verify at their servers that a review comes from Publons, and have an address that vouch after this verification is done.
+Different actors can have different strategies to verify a review. These actors can register their addresses as endorsers, and register a URL explaining how they validate the reviews. For instance, Bloxberg peer-review-app can verify at their servers that a review comes from Publons, and have an address that endorses after this verification is done.
 
-### How to bring trust to public vouchers
-As a decentralized application where we would allow different vouchers, users and applications should decide which vouchers to trust. A system where users could vote up or down specific voucher's addresses could help to build trust in such addresses. Also, we could use the more advanced Token Curated Registries.
+### How to bring trust to public endorsers
+As a decentralized application where we would allow different endorsers. Users, applications and other actors should decide which endorsers to trust. A system where users could vote up or down specific endorser's addresses could help to build trust in such addresses. Also, we could use the more advanced Token Curated Registries as a decentralized reputation system for endorses identities.
```

Comment:
If create a more robust reputation system for establishing trust in endorses, we should describe it thoroughly in the BLIP. This will likely be a critical aspect of a decentralized peer review process.